### DISCUSSION TITLE
2023-05-31 Weekly Meeting 109 Notes/Agenda

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/omigroup#394] [General]
Weekly Meeting 109 is happening on Wednesday, May 31, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

UPDATES:

MSF delegates

Media group

Avatar interop group

GLTF group

World Hops

**Comment by funksoup** [comment]
MSF-delegates (& GLTF update):

- There was an Oversight meeting last week. @funksoup and @aaronfranke attended the meeting on 05/24/2023, 9pm EST. We were able to get the GLTF physics issue on the agenda, and the discussion went really well. 
-- We presented the [GLTF physics page](https://github.com/omigroup/omigroup/discussions/393) which features a table that compares OMI vs MSFT specs and helps distill the most relevant points. (Special thanks to @aaronfranke and [IndieBio](https://twitter.com/indiebio) for page content.) 
**Short link to this page: https://bit.ly/gltfstandard**
-- The goal is to get more voices and feedback in the mix. We (IndieBio, funksoup & aaronfranke) will follow up with emails to specific mail lists.

- @aaronfranke also mentioned that Eoine from Microsoft finally responded to the OMI physics discussion in github and [wrote a very detailed response](https://github.com/KhronosGroup/glTF/pull/2258#issuecomment-1566983651) explaining the logic behind his choices In turn, [@aaronfranke responded in detail](https://github.com/KhronosGroup/glTF/pull/2258#issuecomment-1567363587) as well. 
Full thread: https://github.com/omigroup/omigroup/discussions/394
@aaronfranke pointed out that Eoine's choices (the Microsoft spec) make room for exception cases while the OMI spec is more of a standard/simple implementation, making the argument that the Microsoft spec is unnecessarily complex.

- @Evo suggested a hack-chat around it and @funksoup suggested that we distill the discussion into highlights again and add it to the  [GLTF physics page](https://github.com/omigroup/omigroup/discussions/393) so that we can spread it to the larger community (MSF included) for a poll, i.e., after distributing information, let the community decide. @IndieBio volunteered to help distill the discussion info again.
 
- REMINDER: There is a board election coming up - it's important that we/OMI have a representative on the MSF board, so we can continue to be a voice for independent creators, small orgs, open source and community/grassroots orgs. Elections will be in a few weeks but we can start preparing our community for that now. (There are some documents we will need to fill out). **We want to put one of us on the board (i.e. @funksoup)**. 

- Open Collective status: @mrmetaverse submitted a support request to Open Collective re: switching from a 501c6 fiscal host arrangement to a 501c3 fiscal host arrangement. They replied and asked for clarification, to which @mrmetaverse responded. Awaiting reply.

**Comment by funksoup** [comment]
Media group: 2 videos ready to go (portal crawls & @madjin's work). @evo needs a title card (@funksoup to help). Portal conversation @ AWE will focus on implementation and calls for participation. Engage with and invite community. 
From @mrmetaverse: what are some ways we can request collaboration or support from others and other groups?
@Evo - Maybe this wants to move into an exploratory group?
@IndieBio - what sorts of community? @evo - working groups i.e., the GLTF issue/working group 
What is the best way to engage? 
@aaronfranke - next GLTF meeting is Thursday, 5:30pm EST
@madjin - cultivating notes in immersive meeting spaces, help to visualize and make work more accessible
@IndieBio - learn by doing, specify types of community: technical implementers or newcomers etc?
@funksoup - suggestion to focus on two types of community: community of practitioners and newcomers. Create events for both: learn by doing/workshop events, world hops (which target both), etc. Maybe brainstorm? Take discussion offline - @indieBio will post in OMI experiments thread to continue.

@madjin - suggested: https://sketchfab.com/3d-models/3d-skill-role-badges-and-pins-e3329ed59b874aad98586657a5f11630
badges for different skills / roles could be a nice way to cultivate birds of a feather

@Jimmy6DOF - mapping the metaverse - any interest, ping him.

**Reply by aaronfranke** [reply]
Note: The GLTF meetings are every Thursday, same time of day :)

**Comment by funksoup** [comment]
@madjin - pointed out the video: "Interoperable Characters / Avatars Exploratory Group - Guest Speakers"
https://www.youtube.com/watch?v=qauaUiv-1JA 

[![MSF_slide_video_speakers](https://github-production-user-asset-6210df.s3.amazonaws.com/34685781/242347896-030936fa-b7ba-4586-bc5e-d99f95f57f25.png)](https://www.youtube.com/watch?v=qauaUiv-1JA)

Metaverse Standards Forum's Interoperable Characters / Avatars Exploratory Group have had a numerous guest speakers over the Spring of 2023. Presenting on this video are: 

[0:00:07](https://www.youtube.com/watch?v=qauaUiv-1JA&t=7s) Avatar LOD - Jin, OMI
[0:40:44](https://www.youtube.com/watch?v=qauaUiv-1JA&t=2444s) Asset Interop USD / glTF Update - Marc Petit, Epic Games
[1:29:25](https://www.youtube.com/watch?v=qauaUiv-1JA&t=5365s) An Introduction to VRM and VRoid Projects - Ytomi, pixiv Inc.
[1:57:36](https://www.youtube.com/watch?v=qauaUiv-1JA&t=7056s) Fashion Wearable Exploratory Group - Aditya Mani, YOLOgram
[2:15:52](https://www.youtube.com/watch?v=qauaUiv-1JA&t=8152s) The SANDBOX: Building the Open Metaverse - Sebastien Borget, The Sandbox
[2:52:05](https://www.youtube.com/watch?v=qauaUiv-1JA&t=10325s) A Brief History of Second Life - Andrew Kertesz, Linden Lab


@madjin also showed a visualization of interconnected Hyperfy worlds:

![viz_portal](https://github.com/omigroup/omigroup/assets/34685781/cacdaa64-949b-4cbd-9625-768e2af7a8af)

![viz_portal_2](https://github.com/omigroup/omigroup/assets/34685781/52dfcedc-d5fe-424b-994b-cdb9ef0c9f93)

Visualization done by [3D Force Graph](https://github.com/madjin/3d-force-graph)

Suggested the following tools:
-  [Infinite Canvas tools](https://infinitecanvas.tools/)
- [Muse app](https://museapp.com/principles/)
- @jimmy6DOF suggested [Softspace](https://twitter.com/yiliu_shenburke/status/1647991232080105472?s=20)
 
**Find ways to more seamlessly do deep work, especially as OMI and MSF communities etc grow. Work more in immersive. Why be limited to linear screen space?**

@madjin's notes: **https://hackmd.io/IGbJgeTKSyipncgTUj5RjQ**

**Comment by funksoup** [comment]
Some water cooler chat:
@JoyceBettencourt - avatarjoy -" love that crossover of gardening/natural science with our metaverse / tech stuff. Right now I am trying to work on propogating an old grape variety."
@IndieBio's talk about metaverse, gaming & metabolism of cities: https://debconf19.debconf.org/talks/111-the-metaverse-gaming-and-the-metabolism-of-cities/
